### ISSUE TITLE
Require a phone number

### ISSUE THREAD

**Issue by jmandel** [inferno-community/fsh-smart-scheduling-links#1]
We should think about a sanity check that would recognize errors like the following (seen in an early implementation)

```
{"system":"phone","value":"vaccine-la-support@[redacted].com"}
```

**Comment by jmandel** [comment]
Related: we should set up a constraint to have a warning if there's not a phone number and a URL provided (even though these aren't strict requirements).

**Comment by okeefm** [comment]
These two things should be addressed by #4. If you want more strict regular expressions for any of those constraints, just let me know!

**Comment by jmandel** [comment]
Thanks -- this looks clean and pragmatic!
